Dockerfile.testkitchen

```diff
@@ -0,0 +1,20 @@
+FROM mesosphere/aws-cli
```

Comment:
Should be stable at this point. They had a major change a couple of years ago, but there's been very little since.